internal/clients/composer/client.go

```diff
@@ -83,15 +83,17 @@ func (cc *ComposerClient) request(method, url string, headers map[string]string,
 	for k, v := range headers {
 		req.Header.Add(k, v)
 	}
-	req.Header.Add("X-External-Id", common.RequestId(req.Context()))
 
 	token, err := cc.tokener.Token(req.Context())
 	if err != nil {
 		return nil, err
 	}
 	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token))
 
-	resp, err := cc.client.Do(req)
+	// TODO context must be passed to the request method
+	req = req.WithContext(context.TODO())
+	doer := strc.NewTracingDoer(cc.client)
+	resp, err := doer.Do(req)
```

Comment:
All the remaining HTTP clients contains the same - the doer wrapper cannot add `trace_id` propagation HTTP header until context is available.